x-pack/plugin/src/test/resources/rest-api-spec/test/security/hidden-index/11_security-6_read.yml

```diff
@@ -47,12 +47,12 @@ teardown:
       catch: forbidden
       headers: { Authorization: "Basic dGVzdF91c2VyOngtcGFjay10ZXN0LXBhc3N3b3Jk" } # test_user
       indices.get:
-        index: ".security-6"
+        index: ".security-7"
```

Comment:
all of these changes will be skipped on backport to `6.7` as the index name will default to `.security-6`